### ISSUE TITLE
try different weight parameters in loss terms

### ISSUE THREAD

**Issue by hasibzunair** [hasibzunair/masksup-segmentation#14]
For this total loss term `loss = alpha * loss1 + beta * loss2`, use `alpha` and `beta` values of:

1. 0.5, 0.5
2. 0.6, 0.4
3. 0.7, 0.3
4. 0.8, 0.2
5. 0.9, 0.1
6. 0.1, 0.9
7. 0.2, 0.8
8. 0.3, 0.7
9. 0.4, 0.6
10. 5, 5
11. 10, 5
12. 20, 5

Make a table showing jaccard and dice scores for each configuration. Make sure to run the notebook https://github.com/hasibzunair/context-invariance-segmentation/blob/master/train_siamseg.ipynb before running this set of experiments. Save all logs for each config.
